### ISSUE TITLE
Timecodes for "ML Zoomcamp Office Hours - Week #8 - Midterm Project"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#163]
Youtube video: https://www.youtube.com/watch?v=3bo7I9LaN7o

**Comment by dimzachar** [comment]
0:01:44 - Introduction to multi-class classification and regression.
0:03:51 - Working with continuous variables and text.
0:06:19 - Building a model to predict neighborhoods.
0:09:01 - Demonstration of train-test split and linear regression.
0:12:16 - Logistic regression trains 3 models internally.
0:14:34 - Training decision tree model for classification.
0:17:41 - Using decision trees for multi-class classification.
0:20:17 - Using correlation to determine numerical feature importance.
0:23:09 - Cutting price into intervals for analysis.
0:25:37 - Equal-sized bins using quantiles in pandas.
0:27:59 - Using mutual information to predict prices.
0:30:41 - Using text for machine learning models.
0:33:09 - Creating binary array for text analysis.
0:35:20 - Explanation of TF-IDF vectorizer technique.
0:37:32 - Using time series data for regression.
0:39:42 - Time-based split for non-shuffled data.
0:41:53 - Hyper parameter tuning using validation dataset.
0:44:18 - Importance of a specific feature and order of data processing  Description: Feature importance and data processing order
0:46:29 - Criteria for evaluation and optional testing
0:48:34 - Import settings for cloud deployment tutorial
0:50:43 - Discussion on project logistics and cross-validation.
0:52:48 - Tips for working with large data sets: use linear models, try stochastic gradient descent classifier, sample data, and be careful with evaluation metrics.
0:55:02 - Expert advice on handling imbalanced data sets.
0:57:25 - Optimizing hyperparameters with Grid Search CV
0:59:37 - Grid search CV for model parameter tuning.

**Comment by amitfrancis** [comment]
Updated timecodes, thanks!
